pkg/controllers/provisioner/v1alpha1/allocation/greedyallocator.go

```diff
@@ -29,7 +30,7 @@ type GreedyAllocator struct {
 	Capacity cloudprovider.Capacity
 }
 
-//
+// Allocate will take a list of pending pods and create node based on resources required.
```

Comment:
pending pods -> equivalently schedulable pods
will take -> takes
create node -> creates nodes

Reply:
Allocator doesn't know if they `equivalently schedulable pods`, not sure if I am missing something here.

Reply:
Ah you're right. The other statements hold.